documents/src/pages/utils/cache.md

```diff
@@ -0,0 +1,133 @@
+<!-- 
+title: Local caching Utility
+location: ./utils/cache
+type: page
+layout: default
+-->
+
+::status-complete::
+
+# Local cache
+The utility provides provides an alternative way to store data on your local machine.
+There are 2 main classes that collaborate: wrapper and storage. LocalCache is high level class that wraps the selected storage to be asynchronous api.
+The second is storage class. Store data on the user's machine. The storages has a list of support as below:
+- localstorage
+- indexeddb
+
+LocalCache can select only one storage to use. Here is a common use.
+
+```typescript
+import { LocalCache } from '@refinitiv-ui/utils/cache.js';
+// Create cache and use the storage
+const cache = new LocalCache(
+  // prefix key for localstorage or database name for indexeddb
+  'my-cache',
```

Comment:
```suggestion
  'cache-prefix',
```

or something better.

Reply:
You are right. I got confused.

Reply:
Updated by changing comment to name of the cache.